youtrack_sdk/helpers.py

```diff
@@ -114,3 +114,15 @@ def custom_json_dumps(obj: Any) -> str:
 
 def obj_to_json(obj: Optional[BaseModel]) -> str:
     return custom_json_dumps(obj_to_dict(obj))
+
+
+class NonSingleValueError(Exception):
+    pass
+
+
+def get_single_value[T](values: Collection[T]) -> T:
+    try:
+        (value,) = values
+    except ValueError as e:
+        raise NonSingleValueError(f"Single element expected, found: {values}") from e
```

Comment:
Why do you want to start an error message with a lowercase letter if we use error messages with a capital letter everywhere?

Reply:
I'm not sure where "we" use it, it could be that I oversaw some, or didn't complain, because sadly there is no linter for it. We have this agreement though:

https://moneymeets.atlassian.net/wiki/spaces/IT/pages/2149875726/Backend+Working+Guidelines

> Start exception messages with a lower case letter since Python outputs errors in the format <error class>: <messages>.

Justification:

```python
>>> raise ValueError("life is hard")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ValueError: life is hard
```

Reply:
> I'm not sure where "we" use it, it could be that I oversaw some, or didn't complain, because sadly there is no linter for it.

Ok, then don't search for the `raise` word in the projects code :)

> We have this agreement though

I missed that. Will update the message.

Reply:
Another good catcher (use case) for our fixit linter

Reply:
> Ok, then don't search for the raise word in the projects code :)

🙈 🙉 🙊